__tests__/util/itinerary.js

```diff
@@ -1,4 +1,6 @@
-import {isTransit, routeComparator} from '../../lib/util/itinerary'
+import coreUtils from '@opentripplanner/core-utils'
+
+import { routeComparator } from '../../lib/util/itinerary'
```

Comment:
Didn't we add route comparator to otp-ui? 🤔

Reply:
We did. It is pending release.